$NetBSD: patch-Makefile.am,v 1.1 2015/09/11 12:35:19 wiz Exp $

=========================================================
This patch removes the hardwired -ldl which is a Linuxism
(see also patch-configure.ac)

This patch has been submitted upstream
http://schismtracker.org/scdev/read/1978/
=========================================================


--- Makefile.am.orig	Sun Jul 05 12:54:41 2015 -0400
+++ Makefile.am	Thu Sep 10 00:35:19 2015 +0200
@@ -393,8 +393,3 @@
 
 schismtracker_DEPENDENCIES = $(files_windres)
 schismtracker_LDADD = $(lib_asound) $(lib_win32) $(SDL_LIBS) $(LIBM)
-
-if ! USE_WIN32
-schismtracker_LDADD += -ldl
-endif
-
